just-install.json

```diff
@@ -2456,13 +2456,46 @@
           "arguments": [
             "{{.installer}}",
             "--quiet",
-            "--norestart"
+            "--norestart",
+            "--wait"
           ],
           "extension": ".exe"
         },
-        "x86": "https://aka.ms/vs/15/release/vs_community.exe"
+        "x86": "https://aka.ms/vs/16/release/vs_community.exe"
       },
-      "version": "latest-2017"
+      "version": "latest-2019"
+    },
+    "visual-studio-professional": {
+      "installer": {
+        "kind": "custom",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "--quiet",
+            "--norestart",
+            "--wait"
+          ],
+          "extension": ".exe"
+        },
+        "x86": "https://aka.ms/vs/16/release/vs_professional.exe"
+      },
+      "version": "latest-2019"
+    },
+    "visual-studio-enterprise": {
+      "installer": {
+        "kind": "custom",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "--quiet",
+            "--norestart",
+            "--wait"
+          ],
+          "extension": ".exe"
+        },
+        "x86": "https://aka.ms/vs/16/release/vs_enterprise.exe"
+      },
+      "version": "latest-2019"
```

Comment:
Note that I don't think this will be considered to be the latest version by the j-i tooling.

If this link will always point to VS 2019, the 2019 should be in the package name. Otherwise, this should probably just be latest.

Reply:
@geek1011 The `16` in the URL is for the 2019 edition. I think that the version should be `2019`.

Reply:
But, the link points to the latest *version* of the 2019 edition, thus `2019` as the version doesn't describe everything.

I think it should be `visual-studio-*-2019` with a version of `latest`.

On a side note, I wish I knew why MS uses `16` for 2019 in all their products.

Reply:
Well this was used for the 2017 version so I thought that it will do as well for 2019 but I get your point.
Since the year will be in the software id, should I reintroduced the 2017?

Reply:
Yes, that would be good. Thanks!

Reply:
Done.